assets/app/view/game/part/large_icons.rb

```diff
@@ -124,7 +124,7 @@ def render_part
             decor = h(:circle, attrs: { r: LARGE_RADIUS, fill: 'white', cx: dx, cy: dy })
 
             show_player_colors = setting_for(:show_player_colors, @game)
-            if (decoration = @game.decorate_marker(icon))
+            if (decoration = @game&.decorate_marker(icon))
```

Comment:
why would game be nil

Reply:
It's nil when using a map route directly: 18xx.games/map/1888